app/controllers/questionnaires_controller.rb

```
@@ -98,4 +195,78 @@ def sanitize_display_type(type)
+
+  def build_item_attributes(item_data, index, existing_item = nil)
+    question_type = canonical_question_type(item_data[:question_type])
+    {
+      txt: item_data[:txt].presence || existing_item&.txt,
+      question_type: question_type,
+      weight: item_data[:weight].presence || existing_item&.weight,
+      seq: item_data[:seq].presence || index + 1,
+      alternatives: normalize_alternatives(item_data[:alternatives]) || existing_item&.alternatives,
+      min_label: item_data[:min_label].presence || existing_item&.min_label,
+      max_label: item_data[:max_label].presence || existing_item&.max_label,
+      break_before: item_data.key?(:break_before) ? ActiveModel::Type::Boolean.new.cast(item_data[:break_before]) : true,
```

RuboCop (1.86.1)

[convention] Line is too long. [121/120]

(Layout/LineLength)